### PR TITLE
Add namespell and oelint-adv configs

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
```
Comment (DaniilKl) [inline pyproject.toml:5]: Should a license be specified here? Actually what is the point of adding this file here?

File: hooks/__init__.py
Comment (DaniilKl) [inline hooks/__init__.py:1]: What is the point of adding this file here?

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+- id: namespell
+  name: Name spellchecker
+  description: Check name spelling (case sensitive)
+  entry: namespell
+  language: python
+- id: somehook
```
Comment (DaniilKl) [inline .pre-commit-hooks.yaml]: Should it be deleted?

**Review by macpijan — COMMENTED** [review]
@PLangowski it's getting boring to keep asking for using pre-commit hooks. Even in the repo with the hooks.

File: .pre-commit-hooks.yaml (hunk 2 above)
Comment (macpijan) [reply .pre-commit-hooks.yaml]: Yes.

File: pyproject.toml (hunk 1 above)
Comment (macpijan) [reply pyproject.toml:5]: Yes.

**Pull request merged**